core/domain/feedback_services.py

```diff
@@ -261,3 +262,27 @@ def enqueue_feedback_message_email_task(user_id):
     taskqueue_services.enqueue_task(
         feconf.FEEDBACK_MESSAGE_EMAIL_HANDLER_URL, {'user_id': user_id},
         feconf.DEFAULT_FEEDBACK_MESSAGE_EMAIL_COUNTDOWN_SECS)
+
+
+def send_feedback_message_email(exploration_id, thread_id, message_id):
+    """Creates"""
+    exploration_rights = rights_manager.get_exploration_rights(exploration_id)
+    feedback_message = {
+        'exploration_id': exploration_id,
+        'thread_id': thread_id,
+        'message_id': message_id
+    }
+
+    for owner in exploration_rights.owner_ids:
+        model = feedback_models.UnsentFeedbackEmailModel.get(
+            owner, strict=False)
+
+        if model:
+            model.feedback_messages.append(feedback_message)
+            model.put()
+
+        else:
+            model = feedback_models.UnsentFeedbackEmailModel(
```

Comment:
Well, you answered my question anyway. What I did not know is that these 2 functions I talked about should be executed atomically. I mean wither function 1 is executing or function 2 both of them should now execute at same time.
But as you said their datastore access are atomic so we can have 2 different functions.
I will have a new commit soon.
And yes get_and_delete function will be part of milestone 5 so I am not implementing it now. I will make changes to google doc.

Thanks